lib/cmwx1/cmwx1.py

```diff
@@ -0,0 +1,364 @@
+# This file is part of the cmwx1 module.
+#
+# Copyright (c) 2013-2021 Ibrahim Abdelkader <iabdalkader@openmv.io>
+# Copyright (c) 2013-2021 Sebastian Romero <s.romero@arduino.cc>
+# Copyright (c) 2021 Arduino SA
+#
+# This work is licensed under the MIT license, see the file LICENSE for details.
+#
+# CMWX1ZZABZ-093 driver for Arduino boards.
+# Note this module runs a stock or a custom Arduino firmware.
+
+from utime import sleep_ms, ticks_ms
+from pyb import UART, Pin
+
+MODE_ABP = 0
+MODE_OTAA = 1
+
+RF_MODE_RFO = 0
+RF_MODE_PABOOST = 1
+
+BAND_AS923 = 0
+BAND_AU915 = 1
+BAND_EU868 = 5
+BAND_KR920 = 6
+BAND_IN865 = 7
+BAND_US915 = 8
+BAND_US915_HYBRID = 9
+
+CLASS_A = "A"
+CLASS_B = "B"
+CLASS_C = "C"
```

Comment:
Let's scope them under the Lora class as you suggested to avoid an import like `from cmwx1 import *`